docker/ubuntu/baseimage/install_lint.sh

```diff
@@ -11,12 +11,13 @@ export PATH=${HOME}/local/bin:$PATH
 #install our own updated shellcheck
 SHELLCHECK_VERSION="v0.7.2"
 SHELLCHECK_URL="https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK_VERSION?}/shellcheck-${SHELLCHECK_VERSION?}.linux.x86_64.tar.xz"
-if ! command -v shellcheck; then
+if [ "$(shellcheck --version)" != "v0.7.2" ]; then
```

Comment:
0.8.0 came out recently

Reply:
I don't want to break CircleCi yet.

If i update this I will have to update all our shell scripts. I will make a seperate PR in a few weeks for this.

Also I do not have docker so i did not test the changes with it. The luarocks local stuff was from travis and i didn't see any reason not to just install it globally, but did not test how docker will handle it.

Reply:
PS: 0.7.2 already breaks the frontend CircleCi build

Reply:
https://github.com/koreader/koreader/pull/8605